invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/tombstone.html

```diff
@@ -45,9 +45,6 @@ <h1 class="ui header inline-block">
       <p>
         <strong>{{ _("Date of removal:") }}</strong> {{ record.ui.tombstone.removal_date_l10n_long }}
       </p>
-      <p>
```

Comment:
this will hide it for every case of reason for removal and the goal is to hide it only in case the reason for removal is spam